### DISCUSSION TITLE
[BUG]? Error! Your password must be at least 8 characters and at least one upper case

### DISCUSSION THREAD

**Discussion by MichaelLantz** [mindsdb/mindsdb#3344] [Get help] [answered]
Greetings!

Wanted to reach out before creating an issue. I've searched throughout the mindsdb repo's and can't seem to find anything. When attempting to create a Mindsdb cloud account I ran into an error when I used a more complex password (24 characters mixing lower case, upper case, numbers and special characters). I was able to get past this by simplifying the password to 12 characters without numbers or special characters.

Cheers!

![image](https://user-images.githubusercontent.com/241893/195438317-7eae782e-0a68-4c0d-bf33-18fb0e7d44d3.png)

**Comment by alejandrovillegas** [comment, marked as answer]
Hi  @MichaelLantz 
Great catch! working on this error now

**Comment by MichaelLantz** [comment]
Thank you kindly! :)
